internal/ingress/controller/config/config.go

```diff
@@ -519,6 +519,11 @@ type Configuration struct {
 	// Default: 503
 	LimitReqStatusCode int `json:"limit-req-status-code"`
 
+	// LimitConnStatusCode Sets the status code to return in response to rejected connections.
+	// http://nginx.org/en/docs/http/ngx_http_limit_conn_module.html#limit_conn_status
+	// Default: 503
+	LimitConnStatusCode int `json:"limit-conn-status-code"`
```

Comment:
You need to set the default, like https://github.com/kubernetes/ingress-nginx/blob/2bf5ce5a0ca7da893efd726de9f1c42935bb9ee9/internal/ingress/controller/config/config.go#L682

Reply:
Added